src/main/java/com/rakovets/course_java_enterprise/solution/web-homework-app/WEB-INF/classes/servlets/Task03.java

```diff
@@ -0,0 +1,21 @@
+package servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/task03")
+public class Task03 extends HttpServlet {
+	@Override
+	protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+		String infoBrowser = req.getHeader("User-Agent");
+		resp.setContentType("text/html");
+		PrintWriter writer = resp.getWriter();
+		writer.write("<head><style type=\"text/css\"> p { color: green; font-size: 27px; font-family: Aria; " +
+				"font-weight: bold; } </style></head><body><p>Hi user using browser: <span style='color: black;'>" +
```

Comment:
Wrong message for user

Reply:
> Wrong message for user

Fixed:
message "Hi user {only name Browser}"